### ISSUE TITLE
[BUG]Observation.component.referenceRange is not accessible via GraphQL

### ISSUE THREAD

**Issue by rmhayek** [Aidbox/Issues#441]
**Describe the bug**
Observation.component.referenceRange is not accessible via GraphQL

```
{
  "errors": [
    {
      "message": "Cannot query field `referenceRange' on type `Observation_component'.",
      "locations": [
        {
          "line": 5,
          "column": 7
        }
      ],
      "extensions": {
        "field-name": "referenceRange",
        "type-name": "Observation_component"
      }
    }
  ]
}
```

**Severity**
High

**Steps to reproduce the behavior:**
1. Create an Observation with a component that contains a reference range.
2. GraphQL query Observation provides access to component but within component, no referenceRange.

**Expected behavior**
Be able to view referenceRange as defined in Observation.component.

**Versions:**
- MacOS 11.6
- Docker version 20.10.8, build 3967b7d

- Aidbox image version:
- stable 60263f2fbf6e ("Created": "2021-11-08T15:59:02.20643498Z"
- edge e281b5c9d77a ("Created": "2021-11-08T17:26:21.358284026Z")

**Additional context**
https://www.hl7.org/fhir/observation.html#Observation.component.referenceRange

**Comment by KGOH** [comment]
query reproducing this behaviour

```graphql
query {
  ObservationList(_count: 5) {
    id
    component {
      referenceRange
    }
  }
}

```

**Comment by KGOH** [comment]
Fixed on :edge v2111.1075381a. Added support of elements type defined as contentRefernce 

Problem was that he aidbox graphql schema builder ignored elements defined as [contentReference](http://www.hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.contentReference) such as [Observation.component.referenceRange](https://www.hl7.org/fhir/observation-definitions.html#Observation.component.referenceRange)
